caas/kubernetes/provider/provider.go

```diff
@@ -116,13 +116,24 @@ func (p kubernetesEnvironProvider) Open(args environs.OpenParams) (caas.Broker,
 		return nil, errors.Trace(err)
 	}
 	broker, err := newK8sBroker(
```

Comment:
Can we add a comment to explain that this first broker is to look for the namespace that corresponds with the juju model name

Reply:
Done & Done